### DISCUSSION TITLE
Moving Videos Between Volumes?

### DISCUSSION THREAD

**Discussion by hfreeman02** [biigle/core#872] [Q&A]
Hello!

Is it possible to move videos between volumes without losing the annotations attached to those videos?

Thanks!

**Comment by mzur** [comment]
Hi, this is not easily possible, sorry. If this is just a one-time thing, please write an email to info@biigle.de with the details and links to the videos. We will have a look if moving the videos is technically feasible. We can't do such a thing on a regular basis, though.
